src/pages/sourceSettings/sourceSettings.tsx

```diff
@@ -121,40 +120,21 @@ class SourceSettings extends React.Component<Props, State> {
     const columns = [
       t('source_details.column.name'),
       t('source_details.column.type'),
-      t('source_details.column.last_contacted'),
+      t('source_details.column.added_by'),
+      t('source_details.column.date_added'),
       '',
     ];
-    const wrapper = src => (
-      <>
-        <DetailsTableItem key={`i-${src.name}-${src.type}`} source={src} />
-      </>
-    );
-    const rows = sources
-      .map((src, ix) => [
-        {
-          cells: rowCell(t, src, () => {
-            showDeleteDialog({
-              name: src.name,
-              type: src.type,
-              onDelete: () => {
-                remove(src.uuid);
-              },
-            });
-          }),
-          isOpen: this.state.expanded.indexOf(`${src.name}-${src.type}`) >= 0,
-          // TODO: Uncomment when bulk delete is available
-          // selected: this.state.selected.indexOf(`${src.name}-${src.type}`) >= 0,
-        },
-        {
-          parent: 2 * ix,
-          cells: [wrapper(src)],
-          // TODO: Uncomment when bulk delete is available
-          // selected: true,
-        },
-      ])
-      .reduce((acc, curr) => {
-        return [...acc, ...curr];
-      }, []);
+    const rows = sources.map((src, ix) => ({
+      cells: rowCell(t, src, () => {
```

Comment:
We should follow naming conventions and use camel case for classes. That is, `rowCell` should be `RowCell`.

It would be ideal if file names also match the class name.  For example, rename:

- `deleteDialog.tsx` as `messageDialogBase` (or rename the class as `DeleteDialog`)
- `detailsTable.styles.tsx` as `sourceTable.styles.tsx`
- `table.tsx` as  `sourceTable.tsx`
- `toolbar.tsx` as `filterToolbar.tsx` (or rename the class as just `Toolbar`)